readme/dev/spec/voice_typing.md

```diff
@@ -1,12 +1,70 @@
 # Voice typing
 
-The Android mobile application supports built-in, offline voice typing via the [Vosk library](https://alphacephei.com/vosk/). Vosk is a speech recognition toolkit that can work on lightweight devices, such as mobile phones.
+The Android mobile application supports built-in, offline voice typing via either the [Whisper.cpp](https://github.com/ggerganov/whisper.cpp) or [Vosk libraries](https://alphacephei.com/vosk/). Vosk is a speech recognition toolkit that can work on lightweight devices, such as mobile phones. Whisper is a more advanced speech-to-text library, but will be slower than Vosk.
 
-## Language models
+## Whisper
 
-Vosk uses pre-trained language models that can be used for automatic speech recognition tasks. These models are trained on large amounts of speech data to convert spoken language into written text. Multiple language models are available per language - lightweight ones, which are suitable for mobile (about 50 MB per model), and large ones which are designed for server-side speech recognition (2 GB+ per model).
+By default, Joplin uses Whisper.cpp for voice typing.
 
-## Downloading the language models
+### Language models
+
+Whisper.cpp provides a number of pre-trained models for transcribing speech in different languages. Both [English-only and multilingual models](https://github.com/openai/whisper?tab=readme-ov-file#available-models-and-languages) are available. The multilingual models support a variety of different languages. Joplin uses the smallest of the multilingual models by default.
+
+### Downloading the models
+
+By default, Joplin downloads Whisper models from [this GitHub repository](https://github.com/personalizedrefrigerator/joplin-voice-typing-test/releases). It's possible to download models from a custom location by changing the **Voice typing language files (URL)** in from the "Note" tab of the configuration screen.
```

Comment:
```suggestion
By default, Joplin downloads Whisper models from [this GitHub repository](https://github.com/personalizedrefrigerator/joplin-voice-typing-test/releases). It's possible to download models from a custom location by changing the **Voice typing language files (URL)** in from the "Note" tab of the configuration screen.
```

It may make sense to change this URL to something under the `joplin` GitHub organization.

Reply:
Yes if we could move them to a repository under github.com/joplin before the final 3.3 release that would be better. If you need access for this please let me know

Reply:
> If you need access for this please let me know

I currently don't have the ability to create new repositories in https://github.com/joplin/.

Reply:
I've added you to the org now and created the repository https://github.com/joplin/voice-typing-models Let me know if you're able to create releases there as maybe I need to add you to the repository too

Reply:
Thanks for creating that!

> maybe I need to add you to the repository too

At present, I don't seem to have permission to commit to or create releases in `joplin/voice-typing-models`.

Reply:
I've changed you to admin, please give it another try